.github/workflows/fern.yml

```diff
@@ -0,0 +1,33 @@
+name: ci
+
+on: [push]
+
+jobs:
+  fern-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Install Fern
+        run:  cd sdk && npm install -g fern-api
+
+      - name: Check Fern API is valid
+        run: fern check
+  
+  publish-sdks: 
```

Comment:
You need to setup a repository secret called `NPM_TOKEN` and make it available to this Github action.

Reply:
You will also need a `FERN_TOKEN` to run this code generation in CI. Dmed you one.